tests/NetEscapades.EnumGenerators.Tests/EnumGeneratorTests.cs

```diff
@@ -134,4 +134,33 @@ internal enum MyEnum
         Assert.Empty(diagnostics);
         return Verifier.Verify(output).UseDirectory("Snapshots");
     }
+
+#if NETCOREAPP3_1_OR_GREATER
```

Comment:
We definitely need to figure out why this doesn't work 🤔 Maybe it just doesn't work in the unit tests, perhaps because we're missing an assembly reference?

Reply:
I have no clue what else should be needed besides `System.ComponentModel.DataAnnotations`

Reply:
Me neither 😂 The question is whether this just doesn't work in the unit tests and works fine in the integration tests. My suspicion is that we need additional references [here](https://github.com/andrewlock/NetEscapades.EnumGenerators/blob/main/tests/NetEscapades.EnumGenerators.Tests/TestHelpers.cs#L21) to make it work, but writing an integration test would be a good bet 🙂

Reply:
When I was debugging that's what I noticed:
The Generator can't see the `Name` values of the `Display` attribute, when the target framework is .NET 4.8.
But, when I switch to .NET Core, the values was present.

Reply:
Added the integration test.